components/datetime/src/provider/gregory.rs

```diff
@@ -3,12 +3,14 @@
 // (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
 
 use std::borrow::Cow;
+use yoke::*;
 
-#[derive(Debug, PartialEq, Clone, Default)]
+#[derive(Debug, PartialEq, Clone, Default, Yokeable, ZeroCopyFrom)]
 #[cfg_attr(
     feature = "provider_serde",
     derive(serde::Serialize, serde::Deserialize)
 )]
+#[yoke(CloningZCF)]
```

Comment:
Nit: I think attributes are usually snake case, like `#[yoke(cloning_zcf)]`

Or maybe make it more specific about the consequences, like: `#[yoke(non_zero_copy)]`

Or make it scary, like: `#[yoke(expensive_clone)]`

Reply:
`cloning_zcf` is good for now. I want it to be clear that it's applying to the ZCF